src/constants/presetAssets.ts

```diff
@@ -0,0 +1,20 @@
+export interface presetAsset {
+  assetCode: string;
+  homeDomain?: string;
+  issuerPublicKey?: string;
+}
```

Comment:
We could move this to `types` file.

Reply:
Done in https://github.com/stellar/stellar-demo-wallet/pull/206/commits/839cc6b2a4d0a8dcb694415ab69c459b5df69f47.